lib/rspec/bash/command/call_configuration_manager.rb

```diff
@@ -0,0 +1,21 @@
+module Rspec
```

Comment:
@mdurban - I think it would be good at some point to rework this to be some pure functions that work similar to Redux, with the state being passed in. I say this b/c there b/c this layer doesn't seem terribly useful other than to organize state.

Reply:
Are you thinking about just keeping a global state and using some kind of pub/sub model?

Reply:
Yeah, something like that to get it to be flatter. Having to pass the managers around like this is less than SOLID. Pulling the state out would help with that part at least